zebrad/src/commands/start.rs

```diff
@@ -186,8 +181,7 @@ impl StartCmd {
             app_version(),
             mempool.clone(),
             read_only_state_service,
-            #[cfg(feature = "getblocktemplate-rpcs")]
-            Some(block_verifier),
+            chain_verifier.clone(),
```

Comment:
Zebra isn't really designed to have multiple instances of the same kind of verifier. It would probably work, but it seems trickier to test and maintain.

Instead, I just cloned the common `ChainVerifier`, and used it here.